addons/transparent-orphans/addon.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Transparent orphan blocks",
+  "description": "Makes any blocks that are orphaned (unused with no parent block) transparent.",
+  "tags": ["editor"],
+  "settings": [
+    {
+      "name": "Transparency",
```

Specify that this is a percent?
```suggestion
      "name": "Transparency (%)",
```